sw/otbn/code-snippets/randomness.s

```diff
@@ -111,11 +110,38 @@ test_urnd:
   /* Read the URND WSR again. */
   bn.wsrr w1, 0x2 /* URND */
 
-  /* Fail the test if the two numbers read from URND are equal. */
+  jal x0, error_checking
+
+/**
+ * Perform the following verifications:
+ *
+ *     w0 != w1  - The two random number read shall be different   
```

Comment:
Would you mind:
- Ending each lin a period.
- Indenting the lines consistent with other asm comments (two spaces).
- Wrap the expressions in backticks i.e. `` ` ``.

Reply:
Thanks! Done!